### ISSUE TITLE
Unable to run/build - Could not find manifest-merger.jar (com.android.tools.build:manifest-merger:26.0.0).

### ISSUE THREAD

**Issue by justin-devs** [meteor/meteor#10470]
Meteor Version: 1.8.0.2
OS: Mac OS X High Sierra

I created a new meteor app and developed on my device using meteor run android-device and it worked fine.
After attempting to install cordova-plugin-appcenter-analytics and getting multiple errors on compiling I removed the cordova plugins but this issues started:
LOG:
```
Errors executing Cordova commands: 
 While running Cordova app for platform Android with options --device:
   Error: Command failed: /Users/justin/Desktop/Dev/Meteor/helloSA/.meteor/local/cordova-build/platforms/android/cordova/run --device
   FAILURE: Build failed with an exception.
   
   * What went wrong:
   A problem occurred configuring root project 'android'.
   > Could not resolve all files for configuration ':classpath'.
   > Could not find manifest-merger.jar (com.android.tools.build:manifest-merger:26.0.0).
   Searched in the following locations:
   https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.0.0/manifest-merger-26.0.0.jar
   > Could not find ddmlib.jar (com.android.tools.ddms:ddmlib:26.0.0).
   Searched in the following locations:
   https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.0.0/ddmlib-26.0.0.jar
   > Could not find dvlib.jar (com.android.tools:dvlib:26.0.0).
   Searched in the following locations:
   https://jcenter.bintray.com/com/android/tools/dvlib/26.0.0/dvlib-26.0.0.jar
   > Could not find common.jar (com.android.tools:common:26.0.0).
   Searched in the following locations:
   https://jcenter.bintray.com/com/android/tools/common/26.0.0/common-26.0.0.jar

```

I have tried redownloading sdkmanager manually and through android studio
I have tried making a new app with the same code all with the same error.

Previous meteor apps also no longer works on this MAC with the same issue.

**Comment by justin-devs** [comment]
I was able to pass all errors upon following this guide https://stackoverflow.com/questions/53945047/could-not-find-manifest-merger-jar-com-android-tools-buildmanifest-merger26-0
But this doesnt seem to be the correct practise since i am editing .meteor > local files
When doing meter reset it breaks again

**Comment by justin-devs** [comment]
Any help??

**Comment by KoenLav** [comment]
@justin-devs for now this seems the right approach to fixing this issue.

It has been resolved in Meteor 1.8.1 as well (I believe), which will probably be released sooner rather than later ;)

**Comment by justin-devs** [comment]
@KoenLav 
Thanks! Much relief now!
Do you perhaps know if the issue is occuring for everyone or only after destroying my instance with incompatible cordova plugins?

**Comment by KoenLav** [comment]
@justin-devs I'm not sure, but we experienced the same issue.

**Comment by justin-devs** [comment]
@KoenLav 
Im not sure if this might be off any help
but I am able to debug on mobile using
`meteor run android-device --release=1.8.1-beta.20`
I believe this to be the best until 1.8.1 drops since no altering .meteor is needed

**Comment by cuongdqpayment** [comment]
I had the same error in:

java -version
java version "1.8.0_102"
Java(TM) SE Runtime Environment (build 1.8.0_102-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.102-b14, mixed mode)

$ gradle -v

------------------------------------------------------------
Gradle 4.10
------------------------------------------------------------

Build time:   2018-08-27 18:35:06 UTC
Revision:     ee3751ed9f2034effc1f0072c2b2ee74b5dce67d

Kotlin DSL:   1.0-rc-3
Kotlin:       1.2.60
Groovy:       2.4.15
Ant:          Apache Ant(TM) version 1.9.11 compiled on March 23 2018
JVM:          1.8.0_102 (Oracle Corporation 25.102-b14)
OS:           Mac OS X 10.13.6 x86_64


cordova build android
Android Studio project detected
ANDROID_HOME=/Users/cuongdq/Library/Android/sdk
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home
studio
Subproject Path: CordovaLib
Subproject Path: app

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find manifest-merger.jar (com.android.tools.build:manifest-merger:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.0.0/manifest-merger-26.0.0.jar
   > Could not find ddmlib.jar (com.android.tools.ddms:ddmlib:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.0.0/ddmlib-26.0.0.jar
   > Could not find dvlib.jar (com.android.tools:dvlib:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/dvlib/26.0.0/dvlib-26.0.0.jar
   > Could not find common.jar (com.android.tools:common:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/common/26.0.0/common-26.0.0.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
/Users/cuongdq/IONIC/speedtest-app/platforms/android/gradlew: Command failed with exit code 1 Error output:
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find manifest-merger.jar (com.android.tools.build:manifest-merger:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.0.0/manifest-merger-26.0.0.jar
   > Could not find ddmlib.jar (com.android.tools.ddms:ddmlib:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.0.0/ddmlib-26.0.0.jar
   > Could not find dvlib.jar (com.android.tools:dvlib:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/dvlib/26.0.0/dvlib-26.0.0.jar
   > Could not find common.jar (com.android.tools:common:26.0.0).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/common/26.0.0/common-26.0.0.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
[ERROR] An error occurred while running subprocess cordova.
        
        cordova build android exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more 
        information.

**Comment by LautaroLorenz** [comment]
same problem here

**Comment by wardeha** [comment]
Is this resolved for anyone? I started getting the exact same error

**Comment by filipenevola** [comment]
As I have multiple Meteor apps running fine on Android I'm going to assume this is fixed or not directly related to Meteor, if you are still facing this problem please open a new issue with details including cordova-plugins file.
